### ISSUE TITLE
FEAT: entity relationship diagrams (ERD)

### ISSUE THREAD

**Issue by Zerotask** [beekeeper-studio/beekeeper-studio#1160]
**Is your feature request related to a problem? Please describe.**
To get a better overview and for database creation / design it's hard to get the big picture.
With visualization it's much better to get information about the database design.

**Describe the solution you'd like**
A simple button / option to show an entity relationship diagram (ERD) for the current selected database.
Example:
![image](https://user-images.githubusercontent.com/20150243/162510000-0833baf1-d4de-4625-b4d8-604001a2dc54.png)

**Describe alternatives you've considered**
Use external tools or don't use it at all.

**Additional context**
Add any other context or screenshots about the feature request here. (we love screenshots with red circles if the request is for something visual)

**Database type, if appropriate**
Mainly for relational databases.

**Comment by rathboma** [comment]
Yes I think this can be helpful. I'm going to mark this for the ultimate edition though because it's a lot of work :-).

**Comment by theOofDoctor** [comment]
@rathboma Hey! I'm actually having to do something similar for work right now, thought I might just plug what I'm using to do it: mermaid-js. Has relatively simple syntax and I'm currently building a system to generate a graph based on input. Idk, could be useful.

**Comment by rathboma** [comment]
Some notes from a university professor on how they use ERDs / reverse engineering features of other tools:

>Yes, some calls this task Reverse Engineering (Navicat does, in pgAdmin it’s just using the ERD tool).
I find the reverse engineered diagram very useful when you want to get an overview of the database. It can also be a good support in checking that the implementation of the model is correct, and in general checking the quality of the data modelling.
In my fall course students are required to model, implement and use a database according to some specific needs and requirements (all work on the same domain). In that connection students are required to do (“forward” engineering) ER modelling, producing a model and describing it in an ER diagram that is part of the hand-in. When the model is implemented they are supposed to do the reverse engineering, revise their implementation if something is wrong or missing, and hand in the reverse engineered diagram as well.
